deploy/deploy_app_server.yml

```diff
@@ -78,6 +78,8 @@
   roles:
     - jdk
     - r
+    - r-core
+    - r-libs
```

Comment:
@lel99999 One last thing to do here, remove the `- r` role from these playbooks, since `r-core` and `r-libs` replaces it.

Reply:
@lel99999 I see that the "r" role was removed below, but it is still in this playbook.  Can you check all playbooks and make sure the `- r` is replaced by `- r-core` and `- r-libs` in each one?  Thanks!

Reply:
grepped app_server, gocd, mesos_agent, research_env ...; should be good to go now